src/Home.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function Home() {
+	return (
+			<h1>congrats this is the home page</h1>
```

Comment:
This formatting is a little funky, but not a blocking problem. Consider dedenting (is that a word?) so it's only indented once compared to line 4